webcompat/static/js/lib/issues.js

```diff
@@ -234,7 +234,7 @@ issues.ImageUploadView = Backbone.View.extend({
     var DELIMITER = '\n\n';
     var textarea = $('.js-Comment-text');
     var textareaVal = textarea.val();
-    var imageURL = _.template('![Screenshot of the site issue](<%= url %>)');
+    var imageURL = _.template('[![Screenshot of the site issue](<%= url %>)](<%= url %>)');
```

Comment:
My comment was more along the lines of `(<%= url %>)](<%= url %>)` -- should the first `url` be the thumb url? But that's probably what you mean by "I haven't fixed it completely yet".